docs/guides/rpc.md

```diff
@@ -523,8 +533,6 @@ const res = await client.posts.$post({
 
 If your project is a monorepo, this solution does fit well. Using a tool like [`turborepo`](https://turbo.build/repo/docs), you can easily separate the server project and the client project and get better integration managing dependencies between them. Here is [a working example](https://github.com/m-shaka/hono-rpc-perf-tips-example).
 
-If your client and server are in a single project, [project references](https://www.typescriptlang.org/docs/handbook/project-references.html) of `tsc` is a good option.
```

Comment:
Should we remove this line? I think it's OK to leave it.

Reply:
I added it above in its section `#### TypeScript project references`. Should I leave it here as well?

Reply:
Ahhhh, I missed it! Let's go with this.